client/src/components/NewDogParkForm.js

```diff
@@ -72,102 +72,101 @@ const NewDogParkForm = props => {
     }
 
     return (
-        <div className="callout">
-            <h1>Add a new Dog Park</h1>
+        <div className="new-park-form">
```

Comment:
consider constraining the width of the form so it doesn't span the entire page. aim for some left and right margins